### ISSUE TITLE
Question - restore to a new fs

### ISSUE THREAD

**Issue by yuriw** [rescuezilla/rescuezilla#159]
Can this tool make a backup of the Ubuntu os running on zfs and be restored to ext4 fs ?

thx

**Comment by shasheene** [comment]
Rescuezilla will make a sector-by-sector backup of a ZFS partition, but this backup will be restored as a ZFS partition.

Rescuezilla is a disk imaging tool that operations on partitions and copies the filesystem structures. It's mainly used for making a backup of an entire PC and restoring an identical copy to another hard drive, or the same hard drive at a later date. It sounds like this is not what you want.

It sounds like you want a **file-based copy** of the contents of a ZFS partition to an EXT4 partition? I recommend an rsync-based tool. `rsync` operates on files and folders, has options to retain metadata and operate recursively and can very easily operate over the network using SSH. There are rsync frontends but I don't know which is best.

**Comment by yuriw** [comment]
@shasheene thank you!

it feels that fs level plus file-based-copy combined in one tool would be great !

**Comment by shasheene** [comment]
I agree including file-based copy feature in Rescuezilla would be useful.

Basic file-based copy will definitely become available through Image Explorer functionality (currently in beta and way too slow to be useful), and a future frontend to TestDisk:

* [task #12 (Create graphical frontend to testdisk)](https://github.com/rescuezilla/rescuezilla/issues/143)
* [task #142 (Add ability to restore (and explore) raw "dd" whole disk images, ISO9660 filesystems, etc)](https://github.com/rescuezilla/rescuezilla/issues/143)

I might also include an frontend to rsync (and potentially link it from the Rescuezilla home menu), but I don't have plans for file-based backup beyond this.

**I am worried about over-complicating the interface, so for sometime the focus will be on the bread-and-butter of Rescuezilla: easy-to-use disk imaging.**

**Comment by megvadulthangya** [comment]
Hi! @shasheene is it possible to add zfs support to rescuezilla live cd...  I mean sometimes would be usefull to just use the live disk to copy or edit files on the zfs partition... for example edit some config when crash... read logs etc ...  Integrate into the pcmanfm ...
Without messing with mount the zfs thingy in terminal...?
